scripts/prepare_compliance_data.py

```diff
@@ -0,0 +1,155 @@
+            shutil.copy(path.join(self.inputDirectory, refBase) + ".json",
+                        path.join(self.hg37Directory, refBase) + ".json")
+
+        # Clean out, make and repopulate dataset directories
+        shutil.rmtree(self.datasetsDirectory, ignore_errors=True)
+        os.makedirs(self.datasetsDirectory)
+
+        for ds in self.datasets:
+            dsdir = path.join(self.datasetsDirectory, ds)
+            os.makedirs(dsdir)
+
+            # Reads
+            print("Converting reads...", file=sys.stderr)
+            dsReadsdir = path.join(dsdir, "reads")
+            os.makedirs(dsReadsdir)
+            for readFile in self.datasetReads[ds]:
+                destFile = path.join(dsReadsdir,
+                                     readFile.split('_')[1].split('.')[0]) \
+                           + ".bam"
+                readSrc = pysam.AlignmentFile(path.join(self.inputDirectory,
+                                                        readFile), "r")
+                readDest = pysam.AlignmentFile(destFile, "wb",
+                                               header=readSrc.header)
+                destFilePath = readDest.filename
+
+                for readData in readSrc:
+                    readDest.write(readData)
+                readDest.close()
+                readSrc.close()
+                pysam.index(destFilePath)
+
+            # Variants
+            print("Converting variants...", file=sys.stderr)
+            dsVariantsdir = path.join(dsdir, "variants")
+            os.makedirs(dsVariantsdir)
+            for vgroup in self.datasetVariants[ds].keys():
+                vgroupdir = path.join(dsVariantsdir, vgroup)
+                os.makedirs(vgroupdir)
+                for variantFile in self.datasetVariants[ds][vgroup]:
+                    destFile = path.join(vgroupdir, variantFile.split('_')[2])
+                    shutil.copy(path.join(self.inputDirectory, variantFile),
+                                destFile)
+                    # Pysam's tabix_index automatically compresses the file
+                    # in place, creates a tabix index.
+                    pysam.tabix_index(destFile, preset="vcf")
+
+    def cleanup(self):
+        print("done converting compliance data.", file=sys.stderr)
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Script to generate data bundle from a locally stored "
+        "(and possibly locally edited) version of the compliance dataset.")
+    parser.add_argument(
+        "--outputDirectory", "-o", default="ga4gh-compliance-data",
+        help="The directory to output the server-ready data bundle to.")
+    parser.add_argument(
+        "--inputDirectory", "-i",
+        help="Path to local directory containing compliance dataset.",
+        default='.')
+    parser.add_argument('--verbose', '-v', action='count', default=0)
+    args = parser.parse_args()
+    cdm = ComplianceDataMunger(args)
+    try:
```

Comment:
I don't see why the try/finally is necessary.  The only cleanup is printing that it's done, which shouldn't be printed if the process fails.